docs/issue_picker.md

```diff
@@ -0,0 +1,31 @@
+## Issue Picker 
+A simple picker that allows you to select issues easily on current opened repository.
+
+### Getting started
+Press <kbd>Ctrl</kbd> + <kbd>I</kbd> to `launch` the issue picker. 
+
+![](images/pickers/issue_picker_main.png?raw=true)
+
+The `Issue Picker` is divided into 3 main sections:
+ 1. `Selected issues` : shows issues that will be selected upon confirmation 
+ 2. `Text field`     : enter your query here to filter all issues in current repository 
```

Comment:
It is a description of the container where users can run their queries. Maybe `Query field`